recipes/diffpy-structure/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "diffpy.structure" %}
+{% set version = "1.3.4" %}
+{% set sha256 = "de481329c4dc41ce1538f88fac768c8f677860602caa886fbfb3429bc17ac3ae" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/diffpy/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Comment:
I see - the filename in the URL should have been `diffpy.Structure-1.3.5.tar.gz`.  I will make sure the tarball for the next release is all lowercase - hopefully PyPI sets this according to uploaded file and not from package name.